### DISCUSSION TITLE
Multiple Issues/questions.  Help requested.

### DISCUSSION THREAD

**Discussion by SplitMilky** [argon-rbx/argon#142] [Q&A] [answered]
1. Is there a more thorough guide? I seem to get things working, no errors on install but there are issues.

2. When I create new scripts or move scripts, argon makes duplicates of the script with in ID  and adds a json file.  If I move one script to another folder, I get another set of duplicates and the previous ones don't go away.
![image](https://github.com/user-attachments/assets/f7cfab2c-528b-4dd1-98b8-4f1499e7a8e1)

3. It appears there is duplicate folders  now as well.

4. I have sync back on but it doesn't seem to work for RemoteEvents in ReplicatedStorage.   Also in VS Code how do I make a new file in ReplicatredStorage so it is recognized as a RemoveEvent

5. When using the Argon menu to launch Roblox with a workspace does nothing, and i see no errors. 

6. Each time i run I get an exception from what I gather is added from Argon? I can skip and play on but it is annoying.
![image](https://github.com/user-attachments/assets/a1d2c2af-ffb2-4adb-8616-5dca4ad4a624)

7. I'll leave it here for now, other things may be the result of these so hopefully we can get some of this stuff sorted first. 

Thank you.

**Comment by DervexDev** [comment, marked as answer]
Hey, sorry for the delayed response but I'm very busy now.

1. Yes, there is [Getting Started](https://argon.wiki/docs/category/getting-started) page on [argon.wiki](https://argon.wiki/). There is also an example place [here](https://argon.wiki/examples/place).
2. It's because of the [`rename_instances`](https://argon.wiki/docs/configuration#global-config) setting. It doesn't work with some client configurations so you might wanna disable it if you don't need it. Alternatively, you can disable `Two-Way Sync` in plugin settings.
3. See the above point.
4. You need to disable `Only Code Mode` in plugin settings, read more [here](https://argon.wiki/docs/configuration#argon-plugin). To create `ReplicatredStorage` in your filesystem you can either use [Data File](https://argon.wiki/api/file-types#data-file) or [JSON Model](https://argon.wiki/api/file-types#json-model).
5. Could you send Argon logs from View->Output->Argon (in VS Code) or just use CLI and copy the output from here. Also, what OS are you using? It's not supported on Linux.
6. It's not Argon.

**Reply by DervexDev** [reply]
Marking this as an answer due to inactivity
